	uint8 dataID
	uint8 operationEnable
	uint8 targetlArmCoreMode
    uint8 Endeffector_open
    int32[6] targetPositions
    int32[6] targetVelocities
	int32[6] targetAcceleration